settings.inc

```diff
@@ -23,6 +23,31 @@ include 'domain.bootstrap.inc';
 // can explode rather maginificently. Using a try/catch, we can at least fail
 // silently.
 try {
+  global $databases, $settings;
```

Comment:
I don't think `$settings` is needed.  I tested without it and didn't notice any difference. But am I missing something?

```suggestion
  global $databases;
```